console/console-init/ui/src/modules/connection-detail/components/ConnectionDetailToolbar/ConnectionDetailToolbar.tsx

```diff
@@ -8,29 +8,28 @@ import {
   PageSection,
   PageSectionVariants,
   Title,
-  Pagination,
   GridItem,
   Grid
 } from "@patternfly/react-core";
 import { GridStylesForTableHeader } from "modules/address/AddressPage";
-import { ConnectionLinksListPage } from "./ConnectionsLinksListPage";
-import { useLocation, useHistory } from "react-router";
+import { ConnectionLinksContainer } from "../../containers";
```

Comment:
As we are not using relative path anymore. please use absolute path as we are following.
It should be look like .
import { ConnectionLinksContainer } from "modules/connection/containers";

Reply:
Done